src/convert.jl

```diff
@@ -112,10 +112,16 @@ end
 function unsafe_convertcrs(::Type{<:GFT.ProjString}, crsref)
     return GFT.ProjString(toPROJ4(crsref))
 end
+function unsafe_convertcrs(::Type{<:GFT.EPSG}, crsref)
```

Comment:
Can we add a test for this method? It seems to not be covered per https://app.codecov.io/gh/yeesian/ArchGDAL.jl/pull/391.

Reply:
This method is overwritten at line 124, so we need to remove the duplicate.

Reply:
now fixed